Source/Tools/FEXQonfig/main.qml

```diff
@@ -603,6 +603,12 @@ ApplicationWindow {
                                     tooltip: qsTr("Controls TSO emulation on memcpy/memset instructions")
                                     config: "MemcpySetTSOEnabled"
                                 }
+                                ConfigCheckBox {
+                                    leftPadding: 24
+                                    text: qsTr("Strict in-process split-lock atomics")
+                                    tooltip: qsTr("Controls split-lock emulation strictness")
```

Comment:
The tooltip doesn't add much. Either it should meaningfully elaborate on the feature, or it should be dropped.

Reply:
Changed the tooltip that it controls if split-locks can tear or not.

Reply:
Does it still need a tooltip if the whole text is changed to "Enable non-tearing split-lock atomics"? Not sure how critical it is to mention them being in-process, since there's no other option that it could be confused with anyway.

Reply:
Sure, removed the tooltip and changed the option name